app/src/main/java/org/mozilla/fenix/compose/Divider.kt

```diff
@@ -0,0 +1,154 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Generic Divider
+ *
+ * @param modifier [Modifier] Used to shape and position the underlying [androidx.compose.material.Divider]
+ * @param color  [Color] of the divider line
+ */
+@Composable
+fun Divider(
```

Comment:
Just wondering what is your plan - we could audit and replace all the existing `Divider` usage with this new component in this PR unless you want to save that as a followup.

Reply:
I wasn't sure if I should replace all the existing `Divider`s in the code in this PR, I was thinking I could refactor in a second one.
if it makes more sense to do it all in this PR, I'll make those changes too

Reply:
I will leave the decision up to you on whether you want to do this here or as a second PR. Just wanted to inquire about your plans. The ask would be to file a followup issue if you chose to do this in a second PR.

Reply:
Included replacing existing usage of `Divider` in Fenix

Notes:
`TabGridItem` had a thickness of 1.dp and this is maintained by the default in `Divider`
`RecentlyVisited` has a new thickness of 1.dp in the figma here: https://www.figma.com/file/am9EJGg6ow29PXWkc9jcOH/MR2-(2021)?node-id=3022%3A48384